src/Voron/Impl/Journal/WriteAheadJournal.cs

```diff
@@ -624,10 +624,8 @@ public void OnTransactionCommitted(LowLevelTransaction tx)
 
             public void OnTransactionCompleted()
             {
-                if (_waitForJournalStateUpdateUnderTx.IsSet == false)
-                {
-                    _onWriteTransactionCompleted.Set();
-                }
```

Comment:
Can you please explain the change made in this commit?

Reply:
Added comments